runelite-client/pom.xml

```diff
@@ -122,7 +122,6 @@
 			<groupId>net.runelite</groupId>
 			<artifactId>orange-extensions</artifactId>
 			<version>1.0</version>
-			<scope>provided</scope>
```

Comment:
This will probably explode if not `provided`

Reply:
It explodes when provided too, not sure what to do about it.